edx_exams/apps/api/v1/views.py

```diff
@@ -388,13 +394,62 @@ def get(self, request, exam_id):
         return response
 
 
+class LatestExamAttemptView(ExamsAPIView):
+    """
+    Endpoint for the fetching a user's latest exam attempt.
+    /exams/attempt
+
+    Supports:
+        HTTP GET: Get the data for a user's latest exam attempt.
+
+    HTTP GET
+    Fetches a user's latest exam attempt.
+
+    **GET data Parameters**
+        'user': The data of the user whose latest attempt we want to fetch.
+
+    **Returns**
+    {
+        'id': int (primary key),
+        'created': datetime,
+        'modified': datetime,
+        'user': User object,
+        'start_time': datetime,
+        'end_time': datetime,
+        'status': string,
+        'exam': Exam object,
+        'allowed_time_limit_mins': int,
+        'attempt_number': int,
+    }
+    """
+
+    def get(self, request):
+        """
+        HTTP GET handler to fetch all exam attempt data
+
+        Parameters:
+            None
+
+        Returns:
+            A Response object containing all `ExamAttempt` data.
+        """
+        user_id = request.user.id
+        attempt = get_latest_attempt_for_user(user_id)
+
+        if attempt is not None:
+            serialized_attempt = ExamAttemptSerializer(attempt)
```

Comment:
Now that #78 has been merged we should be using the StudentAttemptSerializer. This constructs the necessary field names the UI expects